### ISSUE TITLE
Is TTT still maintained?

### ISSUE THREAD

**Issue by igor-cali** [redfish64/TinyTravelTracker#107]
Hi @redfish64!
I wonder if this app is still maintained. I am updating and refreshing it in my fork (e.g. migrating to AndroidX and refreshing some dependencies).
Of course, I am facing some hurdles here and there.
If you do not like to maintain TTT anymore, I would like to publish my fork under a new name. Of course, I would mention your original great work.
Thanks in advance for your comments.

**Comment by Harova** [comment]
I love this project. I didn't find anything even close. Simple, lightweight, efficient, secure, private, foss... I didn't even find an app that does the same (all-time local gps logging).
So far, I've had no problem with the current versions, but I do have an "old" phone (android 9). I worry what would happen when I upgrade the phone.
I'd love to see the project continue. Last update from @redfish64 was a long while ago. I hope he's doing fine. If @redfish64 can't maintain it anymore it would be awesome if you (@igor-cali) can keep it alive. Let us know ;)
Thank you!

**Comment by igor-cali** [comment]
> I'd love to see the project continue. Last update from @redfish64 was a long while ago. I hope he's doing fine. If @redfish64 can't maintain it anymore it would be awesome if you (@igor-cali) can keep it alive. Let us know ;)

@Harova i consider [my fork](https://gitlab.com/igi0/TinyTravelTracker) alpha level. Currently trying it in parallel with TTT on my personal device. Feel free to share your comments there.

**Comment by redfish64** [comment]
Yes, please do. I don't have the time or desire to continue working on this
myself and am glad you are willing to take the reins. Thanks for your work
on this.

**Comment by janexner** [comment]
@igor-cali are you still working on your fork these days?

(I have recently set up a Dawarich instance and I am looking for a good tracker to use with it. I was going to see whether I could add automatic upload to Dawarich to TTT. Just wondering whether I should fork myself, or whether you'd want me to collaborate on your fork, instead.)

**Comment by igor-cali** [comment]
> [@igor-cali](https://github.com/igor-cali) are you still working on your fork these days?
> 
> (I have recently set up a Dawarich instance and I am looking for a good tracker to use with it. I was going to see whether I could add automatic upload to Dawarich to TTT. Just wondering whether I should fork myself, or whether you'd want me to collaborate on your fork, instead.)

Hi @janexner. My fork is on on GitLab --> [here](https://gitlab.com/igi0/TinyTravelTracker). I do not work much on it and I  cannot really test it on other devices, but at least it works on my daily driver phone. You are welcome to contribute to it, if you wish.
